### PR TITLE
Add API Server for organizations

### PR DESCRIPTION
**Pull request by glrf**
This PR adds the deployment of the aggregate API server. 

Depends on https://github.com/appuio/control-api/pull/15

## Checklist

- [x] PR contains a single logical change (to build a better changelog).
- [x] Update the documentation.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.
- [x] Link this PR to related issues or PRs.

### CODE REVIEW

File: component/api-server.jsonnet [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    namespace: params.namespace,
+  },
+
+  spec+: {
+    template+: {
+      spec+: {
+        containers: [
+          if c.name == 'apiserver' then
+            c {
+              image: '%s/%s:%s' % [ image.registry, image.image, image.tag ],
```
Comment (simu) [inline component/api-server.jsonnet:55]: Alternatively:

```suggestion
              image: '%(registry)s/%(image)s:%(tag)s' % image,
```

File: docs/modules/ROOT/pages/how-tos/tls.adoc [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+. Add certificate to cluster config
++
+[source,bash]
+----
+apicert=$(cat apiserver.crt)
+yq eval -i ".parameter.control_api.apiserver.tls.serverCert = \"${apicert}\"" \
+  inventory/classes/${TENANT_ID}/${CLUSTER_ID}.yml
+----
+
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/tls.adoc:52]: I'm missing something along the lines of

```
cd invenctory/classes/${TENANT_ID}
git add ${CLUSTER_ID}.yml
git commit -m "Configure Control API APIService certificate"
git push origin master
```

File: docs/modules/ROOT/pages/references/parameters.adoc [hunk 3]
```diff
@@ -10,6 +10,33 @@ default:: `appuio-control-api`
...
 
+== `apiserver.apiservice`
+type:: dict
+
+This key is applied directly as a `APIService` object.
+The `APIService` object instructs the Kubernetes API server to delegate requests to the aggregate API server.
+
+== `apiserver.tls`
+
+This key configures encryption of internal traffic, meaning from the Kubernetes API server to our aggregate API sever.
```
Comment (simu) [inline docs/modules/ROOT/pages/references/parameters.adoc:21]:
```suggestion
This key configures encryption of internal traffic from the Kubernetes API server to the aggregate API sever.
```

**Review by simu — APPROVED** [review]
LGTM overall, some small suggestions inline

File: tests/golden/insecure/control-api/control-api/01_api_server/01_role.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: control-api
```
[the remark this answers is not in the document]
Comment (glrf) [reply tests/golden/insecure/control-api/control-api/01_api_server/01_role.yaml:5]: Fair enough. I'll need to update the source in https://github.com/appuio/control-api for that.

**Pull request merged**